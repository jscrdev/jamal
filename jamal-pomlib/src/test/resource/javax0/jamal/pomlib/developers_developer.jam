{@import res:javax0/jamal/pomlib/pom.jim}{@comment

This file tests that the developer macro is redefined if the developers macro is used.

}{developer :Peter Verhas:peter@verhas.com}
{developers}
  {developer :Peter Verhas:peter@verhas.com}
  {developer :test developer:test_developer_email@some.domain.com}
{end developers}