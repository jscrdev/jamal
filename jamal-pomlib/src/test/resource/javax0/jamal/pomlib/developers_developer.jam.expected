<developers>
    <developer>
        <name>Peter Verhas</name>
        <email>peter@verhas.com</email>
    </developer>
</developers>
<developers>
      <developer>
        <name>Peter Verhas</name>
        <email>peter@verhas.com</email>
    </developer>
      <developer>
        <name>test developer</name>
        <email>test_developer_email@some.domain.com</email>
    </developer>
</developers>